### ISSUE TITLE
HTTPS / SSL support

### ISSUE THREAD

**Issue by simplenotezy** [northosts/wuxt#113]
I notice the code in functions.php includes references to `https://localhost:3000/` but I notice I cannot access the site using `https`. How can I enable local SSL for the site?

**Comment by northosts** [comment]
That might be a bug, https shouldn't be required for local development and wasn't implemented, will try to check that out as soon as possible. Where did you find these urls?

**Comment by simplenotezy** [comment]
In the `set_headless_preview_link` function on line 52

**Comment by northosts** [comment]
Yes, i see. Will fix this shortly. You just can replace https:// with http://

**Comment by simplenotezy** [comment]
OK great @northosts
